README.md

```diff
@@ -1,100 +1,151 @@
-pip[3] list
+$ source $(poetry env info --path)/bin/activate
 ```
 
-Now, your environment is ready to test the modules within the application.
+Note: If you don't want to create a `venv` virtual environment, `poetry` will create a virtual environment for you. To use `poetry`'s virtual environment, you can skip this step.
+
+### Install Dependencies
 
-Once you have finished testing the application within the virtual environment and want to deactivate it, simply run:
+Running the following command reads the `pyproject.toml` file from the current project, resolves the dependencies and install them:
 
 ```bash
-deactivate
+$ poetry install
 ```
 
-To run any of the example file(s), go to your root directory and execute/run python script in the following way:
+### Obtain Credentials File(s)
 
-Let's say you want to run the `metadata_usage` example - then do this:
+Download a copy of the `credentials.json` file stored on Synapse by running the following command:
 
 ```bash
-python[3] examples/metadata_usage.py
+$ synapse get syn21088684
 ```
 
-### Configure Synapse Credentials
+The `credentials.json` file is required when you are using [`OAuth2`](https://developers.google.com/identity/protocols/oauth2) to authenticate with the Google APIs.
+
+Steps involved in the `OAuth2` [authorization flow](https://github.com/Sage-Bionetworks/schematic/blob/develop/schematic/utils/google_api_utils.py#L18):
 
-Download a copy of the `credentials.json` file (or the file needed for authentication using service account, called `quickstart-1560359685924-198a7114b6b5.json`) stored on Synapse, using the synapse client command line utility. The credentials file is necessary for authentication to use Google services/APIs. To do so:
+- The app (i.e., _schematic_) looks for a `token.pickle` file first. It is a file that is created when you have authorized your account to access Google APIs for the first time. If the file already exists, then it get the credentials from it, meaning you do not have to allow access several times.
+- If the `token.pickle` file does not exist, the app will load the credentials for you from the `credentials.json` file and create a `token.pickle` in your project root directory when you access any of the schematic services (e.g., through the Command Line Interface) and the authorization flow completes for the first time. For this:
+  - A URL will be prompted to you when you access a _schematic_ CLI utility. Copy the URL from the console and open it in your browser.
+  - If you are not already logged into your Google account, you will be prompted to log in. If you are logged into multiple Google accounts, you will be asked to select one account to use for the authorization.
+  - You might see a "Google hasn’t verified this app" warning from Chrome on your browser. To resolve this, click on "Advanced" and then "Quickstart (unsafe)". Select a Google account and authorize the _Quickstart_ script to access Google Drive/Sheets under your account.
```

Comment:
This section contains a lot of details that aren't necessary for getting up and running. Maybe these can be moved to a `details.md` document and you can link there with something like "Click here for more info". 

I prefer concise numbered steps like what I wrote [here](https://lcr-modules.readthedocs.io/en/latest/for_users.html#getting-started-user) for another package I authored.

Reply:
You can see in that link that the "Getting Started" section at the top is just single sentences and commands to run. Subsequent section go into detail about everything (and I have links for users to easily jump around).

Reply:
Makes sense. I've added a `details.md` where we can expand upon topics which need to be addressed in slightly more detail.